features/loopMessages/messages.js

```diff
@@ -0,0 +1,12 @@
+module.exports = [
```

Comment:
I am using a JS file instead of JSON because of the comments that are useful with `channelId` and the template strings for the multi-line messages